ase/rtl/CMakeLists.txt

```diff
@@ -0,0 +1,153 @@
+## Copyright(c) 2017, 2018, Intel Corporation
+##
+## Redistribution  and  use  in source  and  binary  forms,  with  or  without
+## modification, are permitted provided that the following conditions are met:
+##
+## * Redistributions of  source code  must retain the  above copyright notice,
+##   this list of conditions and the following disclaimer.
+## * Redistributions in binary form must reproduce the above copyright notice,
+##   this list of conditions and the following disclaimer in the documentation
+##   and/or other materials provided with the distribution.
+## * Neither the name  of Intel Corporation  nor the names of its contributors
+##   may be used to  endorse or promote  products derived  from this  software
+##   without specific prior written permission.
+##
+## THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+## AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,  BUT NOT LIMITED TO,  THE
+## IMPLIED WARRANTIES OF  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+## ARE DISCLAIMED.  IN NO EVENT  SHALL THE COPYRIGHT OWNER  OR CONTRIBUTORS BE
+## LIABLE  FOR  ANY  DIRECT,  INDIRECT,  INCIDENTAL,  SPECIAL,  EXEMPLARY,  OR
+## CONSEQUENTIAL  DAMAGES  (INCLUDING,  BUT  NOT LIMITED  TO,  PROCUREMENT  OF
+## SUBSTITUTE GOODS OR SERVICES;  LOSS OF USE,  DATA, OR PROFITS;  OR BUSINESS
+## INTERRUPTION)  HOWEVER CAUSED  AND ON ANY THEORY  OF LIABILITY,  WHETHER IN
+## CONTRACT,  STRICT LIABILITY,  OR TORT  (INCLUDING NEGLIGENCE  OR OTHERWISE)
+## ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
+## POSSIBILITY OF SUCH DAMAGE.
+
+cmake_minimum_required(VERSION 2.8.12)
+project("opae-c-ase-platform")
+
+# find afu_sim_setup
+find_program(AFU_PLATFORM_CONFIG
+  NAMES
+  afu_platform_config
+  PATHS
+  /usr/bin
+  /usr/local/bin
+  /opt/local/bin
+  DOC "AFU platform configuration utility")
+
+# Directories
+set(ASE_SERVER_RTL     ${PROJECT_SOURCE_DIR})
+set(ASE_SCRIPTS_IN     ${PROJECT_SOURCE_DIR}/../in)
+
+set(PLATFORM_SHARE_DIR ${CMAKE_SOURCE_DIR}/../platform)
```

Comment:
If you need any of these directories you're doing something wrong. afu_platform_config should be pointing at them when needed.

Reply:
im using them as base search for other files it is ok